spezi-invitation-code/index.js

```diff
@@ -0,0 +1,72 @@
+//
+// This source file is part of the StudyApplication based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+const admin = require("firebase-admin");
+const {https} = require("firebase-functions/v2");
+const {FieldValue} = require("firebase-admin/firestore");
+
+class InvitationCodeVerifier {
+  constructor(firestore) {
```

Comment:
I think it would make sense to make the invitation code path (defaults to `invitationCodes`) and the user path (defaults to `users`) something that is configurable on the invitation code verifier level.